tests/test_integration.py

```diff
@@ -74,12 +72,23 @@ class Meta:
 
 class TestSwagger(unittest.TestCase):
 
-    def test_swagger_valid(self):
+    def test_swagger_without_cache(self):
+        # Test if Swagger API documentation loads
+        # when cache all requests is disabled
+        os.environ['CACHE_ALL_REQUESTS'] = 'False'
```

Comment:
For these values, you don't need the quotes - we want the actual `True` and `False` value, not the string representation.

Furthermore, `'False'` will actually evaluate to `True` because it is not an empty string.  You can see more information about this in the [Python documentation](https://docs.python.org/3/library/stdtypes.html#truth-value-testing).

Reply:
**os.environ['CACHE_ALL_REQUESTS'] = False**
The above expression is evaluating a string against a boolean value. I ran into a **TypeError** during testing. So i changed the  boolean value to a string. so that we are comparing two strings

Or i could have the statement like this may be:
**os.environ['CACHE_ALL_REQUESTS'] is False**